i18n/translations/en.json

```diff
@@ -18,9 +18,11 @@
 	"collapse.all": "Collapse All",
 	"color": "Color",
 	"coordinates": "Coordinates",
+	"copied": "Copied",
```

Comment:
`Problem:` I think the naming is not so great if we also have `copy.success` with very similar content. In theory, the new line should be `copy.success` and the old `copy.succcesses` or so.

Reply:
I've refactored it to use `copy.success` and `copy.success.multiple`